tests/behat/features/insert-a-link.feature

```diff
@@ -13,71 +13,73 @@ So that I can link to a external website or a page on my site
     And I go to "/admin/pages"
     And I click on "About Us" in the tree
 
-  Scenario: I can link to an internal page
-    Given I select "awesome" in the "Content" HTML field
-    And I press the "Insert Link" HTML field button
-    When I select the "Link to a page on this site" radio button
-    And I fill in the "internal" dropdown with "Home"
-    And I fill in "my desc" for "Link description"
-    And I press the "Insert" button
-    # TODO Dynamic DB identifiers
-    Then the "Content" HTML field should contain "<a title="my desc" href="[sitetree_link,id=1]">awesome</a>"
-    # Required to avoid "unsaved changes" browser dialog
-    Then I press the "Save draft" button
+#  Scenario: I can link to an internal page
```

Comment:
Please leave a comment with a due date to restore these tests. I highly dislike commenting out code!